### PR TITLE
String tweaks for machine configuration and windows customization

### LINKED ISSUE #2682: String tweaks for machine configuration and windows customization
### Dev Home version

0.1301.472.0

### Windows build number

_No response_

### Other software

_No response_

### Steps to reproduce the bug

On Machine configuration page, WinGet needs a capital G and there should be a period after the first subtitle:
image

On Windows customization page, Insights should have a lowercase I and the comma after "all things" should be removed:
image

### Expected result

_No response_

### Actual result

see above pics

### Included System Information

_No response_

### Included Extensions Information

_No response_

### PR DESCRIPTION
**Pull request by krschau**
## Summary of the pull request
On Machine configuration page, WinGet needs a capital G and there should be a period after the first subtitle
On Windows customization page, Insights should have a lowercase I and the comma after "all things" should be removed

## PR checklist
- [ ] Closes #2682

### CODE REVIEW

File: tools/Customization/DevHome.Customization/Strings/en-us/Resources.resw [hunk 1]
```diff
@@ -126,15 +126,15 @@
     <comment>Dev drive size free</comment>
   </data>
   <data name="DevDriveInsightsCard.Description" xml:space="preserve">
-    <value>All things, Dev Drives, optimizations, etc.</value>
+    <value>All things Dev Drives, optimizations, etc.</value>
```
Comment (dkbennett) [inline tools/Customization/DevHome.Customization/Strings/en-us/Resources.resw:129]: "All things Dev Drives" seems awkward to me. Maybe "All things Dev Drive" ?

**Review by dkbennett — APPROVED** [review]

File: tools/Customization/DevHome.Customization/Strings/en-us/Resources.resw (hunk 1 above)
Comment (krschau) [reply tools/Customization/DevHome.Customization/Strings/en-us/Resources.resw:129]: @cinnamon-msft ?

Comment (cinnamon-msft) [reply tools/Customization/DevHome.Customization/Strings/en-us/Resources.resw:129]: Yeah "All things Dev Drive" sounds good - I don't own this feature but I agree it sounds better. :)

Comment (krschau) [reply tools/Customization/DevHome.Customization/Strings/en-us/Resources.resw]: Updated.

**Review by EricJohnson327 — APPROVED** [review]

**Pull request merged**